anylink/models.py

```diff
@@ -75,22 +72,6 @@ class AnyLinkModelBase(ModelBase):
     """
     def __new__(cls, name, bases, attrs):
         new_class = ModelBase.__new__(cls, name, bases, attrs)
-
-        # six.with_metaclass() inserts an extra class called 'NewBase' in the
-        # inheritance tree: Model -> NewBase -> object. But the initialization
-        # should be executed only once for a given model class.
-
-        # attrs will never be empty for classes declared in the standard way
-        # (ie. with the `class` keyword). This is quite robust.
-        if name == 'NewBase' and attrs == {}:
```

Comment:
Are you sure this can be dropped? six.with_metaclass is not removed, isn't it?

Reply:
That's correct. But as far as I can tell, the `if` statement can still be removed, because for Django > 1.7 we don't do anything here anyway.